specification/purview/data-plane/Azure.Analytics.Purview.DevopsPolicies/preview/2022-11-01-preview/purviewDevopsPolicy.json

```
@@ -0,0 +1,611 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview devops policies REST API Document",
+    "version": "2022-11-01-preview"
+  },
+  "paths": {
+    "/devopsPolicies/aggregate": {
+      "get": {
+        "tags": [
+          "DevopsPolicy"
+        ],
+        "description": "List all devops policies.",
+        "operationId": "DevopsPolicy_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DevopsPolicyList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "values"
+        },
+        "x-ms-examples": {
+          "DevopsPolicy_List": {
+            "$ref": "./examples/DevopsPolicy_List.json"
+          }
+        }
+      }
+    },
+    "/devopsPolicies/sqlsecurityAuditorRoles/{purviewResourceName}": {
```

I think it would be more "RESTful" to rearrange this path slightly:
```suggestion
    "/devopsPolicies/{purviewResourceName}/sqlsecurityAuditorRoles": {
```

The rationale is that "sqlPerfMonitorRoles" appears to be a "singleton child resource" -- essentially a complex property -- of /devopsPolicies/{purviewResourceName}.

This change also makes clear that "purviewResourceName" is the name of an _existing_ resource, not the new resource that you are creating.

---

Updated.